4 reports.sql

```diff
@@ -0,0 +1,42 @@
+--The reports I need are as following:
+-- 1) How many people are booked per flight as: count of people, flight number, departure airport, when the flight is departing and destination.
+select p.FlightNumber, P.DepartureAirport, p.TimeDeparting, p.ArrivalAirport, AmountOfPassengers = count(*)
+from Passenger p 
+group by p.FlightNumber, P.DepartureAirport, p.TimeDeparting, p.
+
+-- 2) Who isn't checked in for flights departing in the next week, in order to send them reminders to check in.
+select * 
+from Passenger p 
+where PassportIssueDate is null 
+and TimeDeparting < dateadd(week, 1, getdate())
+
+-- 3) How many flights are departing per day, and num of passengers we have on those flights.
+--was it possible to do this in one report?
```

Comment:
It depends how you understood the question. If we are counting and passengers on the flights for a specific day, this would work:  
```
select p.TimeDeparting, Flights = count(distinct p.FlightNumber), Passengers = count(p.PassengerID)
from Passenger p 
group by p.TimeDeparting
```

Reply:
adjusted